### PR TITLE
fix: wrong text wrap

### PR DESCRIPTION
**Pull request by hyoban**
### Linked Issues

https://discord.com/channels/1243823539426033696/1282819783691079734

### Additional context

Merge after #315

### CODE REVIEW

File: src/renderer/src/lib/utils.ts [hunk 1]
```diff
@@ -240,3 +240,7 @@ export const getUrlIcon = (url: string, fallback?: boolean | undefined) => {
 }
 
 export const isEmptyObject = (obj: Record<string, any>) => Object.keys(obj).length === 0
+
+export const isTextWrappedInHtmlTag = (text: string) => {
```
Comment (Innei) [inline src/renderer/src/lib/utils.ts:244]: Why not use CSS, `break-words`?

Comment (hyoban) [reply src/renderer/src/lib/utils.ts:244]: Apply `break-words` to? When the content is text only, it seems `break-words` for prose do not work.

Comment (Innei) [reply src/renderer/src/lib/utils.ts:244]: The `break-words` in tailwind are actually word-wrap, not word-break.

Try to use normal CSS for test

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Comment (hyoban) [reply src/renderer/src/lib/utils.ts]: Oh yes. Updated.

**Review by Innei — APPROVED** [review]

**Pull request merged**